### PR TITLE
Remove DNA on metroids and add pickups

### PR DESCRIPTION
**Pull request by ThanatosGit**

### CODE REVIEW

File: open_samus_returns_rando/files/templates/metroid_template.lua [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            GUI.LaunchMessage("Oops 1" .. spawnGroupName, "Metroid.Dummy", "")
+            return
+        end
+
+        Scenario.WriteToBlackboard("entity_" .. spawnGroupName .. "_dead", "b", true)
+        Scenario.WriteToBlackboard("entity_" .. spawnGroupName .. "_deaths", "i", 1)
+        Scenario.WriteToBlackboard("entity_" .. spawnGroupName .. "_enabled", "b", false)
+        Scenario.WriteToBlackboard("entity_" .. "Alpha_001" .. "_dead", "b", true)
+        Scenario.WriteToBlackboard("entity_" .. "Alpha_001" .. "_deaths", "i", 1)
+        Scenario.WriteToBlackboard("entity_" .. "Alpha_001" .. "_enabled", "b", false)
```
Comment (dyceron) [inline open_samus_returns_rando/files/templates/metroid_template.lua:23]: Was this supposed to be changed?

Comment (ThanatosGit) [reply open_samus_returns_rando/files/templates/metroid_template.lua:23]: no :(

Comment (dyceron) [reply open_samus_returns_rando/files/templates/metroid_template.lua:23]: I don't get its purpose then

Comment (ThanatosGit) [reply open_samus_returns_rando/files/templates/metroid_template.lua:23]: Well, I just tested stuff and things and forgot to remove it

Comment (dyceron) [reply open_samus_returns_rando/files/templates/metroid_template.lua:23]: Ah okay so not intentional

Comment (ThanatosGit) [reply open_samus_returns_rando/files/templates/metroid_template.lua]: Removed it

**Review by dyceron — APPROVED** [review]
The multi room Gammas are likely correct since the group names match. I guess each room has its own spawn group. I assume disablimg one disables the others but I'm not sure. We'll find out later I guess

**Pull request merged**